Source/WebCore/testing/Internals.cpp

```diff
@@ -4362,8 +4380,10 @@ void Internals::setUsesOverlayScrollbars(bool enabled)
 
 void Internals::forceAXObjectCacheUpdate() const
 {
-    if (RefPtr document = contextDocument())
-        document->axObjectCache()->performDeferredCacheUpdate(ForceLayout::Yes);
+    if (RefPtr document = contextDocument()) {
+        if (auto* cache = document->axObjectCache())
```

Comment:
I think this needs to be a smart pointer too for our new safer C++, not sure which one is right for `axObjectCache`.

Reply:
OK, it seems CheckedPtr is the one we should use for non-refcounted things: https://github.com/WebKit/WebKit/wiki/Safer-CPP-Guidelines#when-calling-a-non-trivial-member-function-on-an-object-hold-a-smart-pointer-to-the-object-on-the-stack